### ISSUE TITLE
Intermittent GenericContainerRuleTest::sharedMemorySetTest test failures in CI

### ISSUE THREAD

**Issue by rnorth** [testcontainers/testcontainers-java#1688]
See Azure CI reports for a summary: dev.azure.com/testcontainers/testcontainers-java/_test/analytics?definitionId=5&contextType=build - this test has a ~75% pass rate.

Apparent on Azure CI and Travis CI.

We should investigate why this is a flaky test and resolve.

Error message:

```
Gradle Test Executor 2 > org.testcontainers.junit.GenericContainerRuleTest > sharedMemorySetTest FAILED
    org.testcontainers.containers.ContainerLaunchException: Container startup failed
        at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:290)
        at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:272)
        at org.testcontainers.junit.GenericContainerRuleTest.sharedMemorySetTest(GenericContainerRuleTest.java:409)

        Caused by:
        org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
            at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)
            at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:283)
            ... 2 more

            Caused by:
            org.testcontainers.containers.ContainerLaunchException: Could not create/start container
                at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:350)
                at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:285)
                at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
                ... 3 more

                Caused by:
                org.rnorth.ducttape.TimeoutException: Timeout waiting for result with exception
                    at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:54)
                    at org.rnorth.ducttape.unreliables.Unreliables.retryUntilTrue(Unreliables.java:100)
                    at org.testcontainers.containers.startupcheck.StartupCheckStrategy.waitUntilStartupSuccessful(StartupCheckStrategy.java:35)
                    at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:325)
                    ... 5 more

                    Caused by:
                    java.lang.RuntimeException: Not ready yet
```

**Comment by rnorth** [comment]
This may just be a plain race condition in the test - we seem to have neither a command nor a startup strategy in place, so depending on timing we're at risk of random failure.

Will try and fix in #1680

**Comment by rnorth** [comment]
As indicated above, I believe this was fixed in #1680, commit df65ac499e4690437551e95eda0889f5dee40d85 - we've not seen any further failures for this test in CI.
